.github/workflows/build.yml

```diff
@@ -100,7 +100,42 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-  UIAutomation:
+  VCWallet-e2e:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: Run Wallet Web UI automation
+        run: |
+          echo '127.0.0.1 testnet.orb.local' | sudo tee -a /etc/hosts
+          echo '127.0.0.1 vcwallet.trustbloc.local' | sudo tee -a /etc/hosts
```

Comment:
We need to update doc as well https://github.com/trustbloc/wallet/blob/main/docs/build.md#connections